.flake8

```diff
@@ -62,6 +62,12 @@ ignore =
   # Pickle and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue.
   S301
 
+  # Ignore words from requests, like unprocessable
+  SC200
+
+  # Ignore hardcoded tokens for tests
+  S106
```

Comment:
This is probably a good one to ignore. You can instead do:

```python
 # noqa: S106
```

on the line where you have a "token", but this is probably fine. No action needed.